adabot/lib/common_funcs.py

```diff
@@ -91,22 +91,31 @@ def parse_gitmodules(input_text):
         results.append((submodule_name, submodule_variables))
     return results
 
-def get_bundle_submodules():
-    """Query Adafruit_CircuitPython_Bundle repository for all the submodules
-    (i.e. modules included inside) and return a list of the found submodules.
+def get_bundle_submodules(bundle):
+    """Query Adafruit_CircuitPython_Bundle or CircuitPython_Community_Bundle
+    repository for all the submodules (i.e. modules included inside) and
```

Comment:
```suggestion
    """Query bundle repository for all the submodules (i.e. modules included inside) and
```

Reply:
Changes were made according to suggestions, see https://github.com/adafruit/adabot/pull/218/commits/57d59aebac6c3f1abc46b3bc5f9f8ce5da7832fc. Thanks for reviewing